src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/model/entities/RestrictivePatient.kt

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.model.entities
+
+import org.springframework.data.annotation.CreatedBy
+import org.springframework.data.annotation.CreatedDate
+import org.springframework.data.jpa.domain.support.AuditingEntityListener
+import java.time.LocalDateTime
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.EntityListeners
+import javax.persistence.Table
+
+@Entity
+@Table(name = "RESTRICTED_PATIENTS")
+@EntityListeners(AuditingEntityListener::class)
+class RestrictivePatient(
```

Comment:
Just checking - is that name correct - it doesn't match the table name?!

Reply:
Ah, no it’s a typo. Good spot.

Reply:
Just checking - is that name correct - it doesn't match the table name?!

Reply:
Good spot will fix in the next PR